test/Concurrency/attr_execution/conversions_silgen.swift

```diff
@@ -152,14 +152,15 @@ public func testCallerToCaller(_ x: nonisolated(nonsending) @escaping () async -
 
```

Comment:
@gottesmm  just out of curiosity, how did you perform all these changes to update the test expectations? were they automated in some manner?